### DISCUSSION TITLE
How Restore 718 PCM 3.1 Back To Stock.

### DISCUSSION THREAD

**Discussion by GAC1492** [Mr-MIBonk/M.I.B._More-Incredible-Bash#238] [Q&A]
How do you restore the PCM (3.1) on a 718 back to stock after applying the Navigation enable? I have the backup in the backup folder on the SD card and have looked at the restore option in MIB, but am unsure which sub option to use to restore the unit (undo the navigation  enable) and to restore the unpatched PCM software.

**Comment by harman-f** [comment]
Wrong PCM model for this solution.

MHI2 --> PCM 4/4.1

**Reply by GAC1492** [reply]
Sorry my mistake I meant PCM 4 POG11.
